scabha/schema_utils.py

```diff
@@ -210,3 +213,39 @@ def clickify_parameters(schemas: Union[str, Dict[str, Any]]):
                 decorator_chain = lambda x,deco=deco,chain=decorator_chain: chain(deco(x))
 
     return decorator_chain
+
+@dataclass
+class SchemaSpec:
+    inputs: Dict[str, Parameter]
+    outputs: Dict[str, Parameter]
+    libs: Dict[str, Any]
+
+def paramfile_loader(paramfiles: List[File], sources: List[File] = [], schema_spec=None, use_cache=False) -> Dict:
```

Comment:
Suggest Union[File|List[File]], since that's how you treat it in the code below. Just to make the function signature more self-explanatory.